src/system_descriptor_data.jl

```diff
@@ -216,6 +216,30 @@ const SYSTEM_CATELOG = [
         raw_data = joinpath(PACKAGE_DIR, "data", "RTS_GMLC"),
         build_function = build_test_RTS_GMLC_sys,
     ),
+    SystemDescriptor(
+        name = "RTS_GMLC_sys",
+        description = "RTS-GMLC Full system from git repo",
+        category = PSITestSystems,
+        raw_data = joinpath(PACKAGE_DIR, "data", "RTS-GMLC-master"),
+        build_function = build_RTS_GMLC_sys,
+        download_function = download_RTS,
+    ),
+    SystemDescriptor(
+        name = "modified_RTS_GMLC_DA_sys",
+        description = "Modified RTS-GMLC Full system for day-ahead simulations",
```

Comment:
add some mention of what the modifications are... maybe "RTS-GMLC Full system for day-ahead simulations with modifications to reserve definitions to improve feasibility"

Reply:
yup will do that